### DISCUSSION TITLE
Is it recommended to delete bundle extension from existing installations?

### DISCUSSION THREAD

**Discussion by IngoBelka** [datenstrom/community#814] [Ask a question] [answered]
According to the [announcement of Yellow 0.8.21](https://github.com/datenstrom/yellow/discussions/813#discussion-4442315) you deleted bundle.php from the basic installation.

Do you recommend, to delete bundle.php from existing installations as well, if there are no special needs for it?

**Comment by schulle4u** [comment]
It depends on how many CSS and JS ressources your website includes. If you have many extensions installed and each has it's own CSS or JS module, you should consider leaving the bundle extension installed. For a small website with only a few extensions the bundler probably has only little impact on yur website's performance. In any case it is completely safe to remove the file if you don't need it. The extension by the way is [still available](https://github.com/annaesvensson/yellow-bundle) and will receive updates, it only has been removed from the standard installation.

**Comment by annaesvensson** [comment, marked as answer]
No, I would keep it. You may want to add the bundle extension to a brand new installation.

**Comment by IngoBelka** [comment]
Ok, I didn't had any problems with bundle extension in the past. Because of that I'will leave it as it is. I can't choose two answers, so I take the shorter on. :-)
